actors/builtin/miner/policy.go

```diff
@@ -71,39 +76,41 @@ var MaxSealDuration = map[abi.RegisteredSealProof]abi.ChainEpoch{
-// The challenge is available this many epochs before the window is actually open to receiving a PoSt.
+// Motivation: A miner can start the WindowPoSt computation without waiting for chain stability. This value cannot be too large since it could compromise the rationality of honest storage (due to WindowPoSt cost assumptions)
 const WPoStChallengeLookback = abi.ChainEpoch(20)
 
-// Minimum period before a deadline's challenge window opens that a fault must be declared for that deadline.
-// This lookback must not be less than WPoStChallengeLookback lest a malicious miner be able to selectively declare
-// faults after learning the challenge value.
+// Filecoin Parameter: Minimum period between fault declaration and the next deadline opening.
+// If the number of epochs between fault declaration and deadline's challenge window opening is lower than FaultDeclarationCutoff,
+// the fault declaration is considered invalid for that deadline.
+// Motivation: This guarantees that a miner is not likely to successfully fork the chain and declare a fault after seeing the challenges.
 const FaultDeclarationCutoff = WPoStChallengeLookback + 50
 
-// The maximum age of a fault before the sector is terminated.
+// Filecoin Parameter: The maximum age of a fault before the sector is terminated.
+// Motivation: This guarantees to clients that a Filecoin miner cannot lose the file for longer than 14 days.
 const FaultMaxAge = WPoStProvingPeriod*14
 
-// Staging period for a miner worker key change.
-// Finality is a harsh delay for a miner who has lost their worker key, as the miner will miss Window PoSts until
+// Filecoin Parameter: Staging period for a miner worker key change.
+// Future improvement: Finality is a harsh delay for a miner who has lost their worker key, as the miner will miss Window PoSts until
 // it can be changed. It's the only safe value, though. We may implement a mitigation mechanism such as a second
 // key or allowing the owner account to submit PoSts while a key change is pending.
+// Motivation: This guarantees that a miner cannot choose a more favorable worker key that wins leader elections.
 const WorkerKeyChangeDelay = ChainFinality
 
-// Maximum number of epochs past the current epoch a sector may be set to expire.
+// Filecoin Parameter: Maximum number of epochs past the current epoch a sector may be set to expire.
 // The actual maximum extension will be the minimum of CurrEpoch + MaximumSectorExpirationExtension
 // and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
 const MaxSectorExpirationExtension = builtin.EpochsInYear
 
-// Ratio of sector size to maximum deals per sector.
+// Filecoin Parameter: Ratio of sector size to maximum deals per sector.
 // The maximum number of deals is the sector size divided by this number (2^27)
 // which limits 32GiB sectors to 256 deals and 64GiB sectors to 512
 const DealLimitDenominator = 134217728
```

Comment:
**Clarification requested:** Where does this come from? What is constraining this to be "256" deals for a 32GiB sector? This means an average of 128MiB per deal?

Reply:
This comes from @whyrusleeping, 256 deals for 32Gib, 512 deals for 64Gib. 
Resolution on relevant PR [here](https://github.com/filecoin-project/specs-actors/pull/632#pullrequestreview-446924587)

Reply:
Unfortunately it is not clear from the issue what is the exact motivation "After discovering discussion with @whyrusleeping elsewhere motivating this approach"